### DISCUSSION TITLE
Where to look for current version of defer/stream spec

### DISCUSSION THREAD

**Discussion by miami-man** [graphql/defer-stream-wg#59] [General]
Hi,

I wanted to know where is the best place to look for the current version of the defer/stream spec. I have been referring to https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md, but I know there are discussions ongoing that could change this and does this get updated when decisions are make or is there somewhere else I need to refer to?

Thanks,
D

**Comment by hwillson** [comment]
@dylanaubrey the best place to look is the spec PR itself (it is kept up to date): https://github.com/graphql/graphql-spec/pull/742

**Reply by miami-man** [reply]
One follow up, the changes in the PR such as adding `incremental` field, how do we know when this is getting introduced into a future version of the defer/stream graphql package? is there an easy way to tie the two together (the spec PR and the graphql npm package)?

**Reply by saihaj** [reply]
The latest implementation does contain it

**Reply by miami-man** [reply]
Hi again... is defer/stream support now is the main graphql package versions? I have been using `16.1.0-experimental-stream-defer.6` and this doesn't have the `incremental` field, but there is not a newer version of the `experimental-stream-defer` packages. Thanks.

**Reply by robrichard** [reply]
the latest defer-stream changes are published in `17.0.0-alpha.2`. The `experimental-stream-defer` tag is not being used any more as the code is now merged the to `main` branch of graphql-js

**Reply by miami-man** [reply]
cheers dude, thanks for the quick reply!

**Reply by benjie** [reply]
@robrichard or someone with access, please can you [`npm deprecate`](https://docs.npmjs.com/cli/v8/commands/npm-deprecate) `16.1.0-experimental-stream-defer.6` with instructions to use `17.0.0-alpha.2` or newer?

**Reply by robrichard** [reply]
I pinged @IvanGoncharov to deprecate

**Reply by robrichard** [reply]
16.1.0-experimental-stream-defer.6 is deprecated now
